### PR TITLE
test: fix e2e

### PR DESCRIPTION
**Pull request by hi-ogawa**
`npm i` was missing for `samples/imba` e2e. I also replaced `.click` with `expect(...).isVisible()` since it looks like unnecessary click is causing UI to glitch sometimes.

### CODE REVIEW

File: test-e2e/basic.test.ts [hunk 1]
```diff
@@ -23,13 +26,14 @@ test('basic', async ({ launch }) => {
...
-  await page.locator(`[title*="mix.test.ts (Failed)"]`).click()
-  await page.locator(`[title*="3/7 tests passed"]`).click()
+  await expect(page.locator(`[title*="pass.test.ts (Passed)"]`)).toBeVisible()
+  await expect(page.locator(`[title*="fail.test.ts (Failed)"]`)).toBeVisible()
+  await expect(page.locator(`[title*="mix.test.ts (Failed)"]`)).toBeVisible()
+  await expect(page.locator(`[title*="3/7 tests passed"]`)).toBeVisible()
 })
 
 test('imba', async ({ launch }) => {
+  await execa('npm', ['i'], { cwd: './samples/imba' })
```
Comment (AriPerkkio) [inline test-e2e/basic.test.ts:36]: Does this fix https://github.com/vitest-dev/vitest-ecosystem-ci/actions/runs/8354143640/job/22867013035 ?

Comment (hi-ogawa) [reply test-e2e/basic.test.ts:36]: Yeah, I think that's the same issue.

Comment (hi-ogawa) [reply test-e2e/basic.test.ts:36]: Hmm, technically if `samples/imba` installed its own Vitest, then it's not testing against ecosystem-ci's Vitest.

Comment (hi-ogawa) [reply test-e2e/basic.test.ts:36]: Superseded by
- https://github.com/vitest-dev/vscode/pull/311

I'll also setup a wrapper script for ecosystem-ci, so we can tweak things while keeping the same script on ecosystem-ci repo.

**Pull request closed without merging**